--- thttp/Makefile.in	2018-08-03 18:30:07.056388617 +0900
+++ thttp/Makefile.in	2018-08-03 18:30:20.516737921 +0900
@@ -34,7 +34,7 @@ BINDIR = @sbindir@
 # Pathname of directory to install the man page.
 MANDIR = @mandir@
 # Pathname of directory to install the CGI programs.
-WEBDIR = $(prefix)/www
+WEBDIR = $(prefix)/www/html
 
 # CONFIGURE: The group that the web directory belongs to.  This is so that
 # the makeweb program can be installed set-group-id to that group, and make
